apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -124,12 +127,28 @@ public CcwRifLoadJob(
       DataSetMonitorListener listener) {
     this.appMetrics = appMetrics;
     this.options = options;
-    this.listener = listener;
     this.s3TaskManager = s3TaskManager;
+    this.listener = listener;
 
     this.dataSetQueue = new DataSetQueue(appMetrics, options, s3TaskManager);
   }
 
+  /** @see gov.cms.bfd.pipeline.sharedutils.PipelineJob#getSchedule() */
+  @Override
+  public Optional<PipelineJobSchedule> getSchedule() {
+    return Optional.of(new PipelineJobSchedule(1, ChronoUnit.SECONDS));
+  }
+
+  /** @see gov.cms.bfd.pipeline.sharedutils.PipelineJob#isInterruptible() */
+  @Override
+  public boolean isInterruptible() {
+    /*
+     * TODO This would be a good enhancement: making this class properly interruptible. This would
```

Comment:
Agree on the TODO, but how would you know where you stopped / are picking up from?  Are RIF file loads idempotent?

Reply:
Very idempotent.